Library/Homebrew/test/cmd/rbenv-sync_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require "cmd/rbenv-sync"
+require "cmd/shared_examples/args_parse"
+
+RSpec.describe Homebrew::Cmd::RbenvSync do
+  it_behaves_like "parseable arguments"
```

Comment:
I wonder if we could just automatically run `it_behaves_like "parseable arguments"` on all abstract command classes. The would remove the need to add this for every command and make it impossible to forget to add this test.